### PR TITLE
Add basic Toronto 2017 site

### PR DESCRIPTION
**Pull request by WhatsChrisDoing**

### CODE REVIEW

File: data/events/2017-toronto.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  - name: "Amy Mansell"
+    twitter: "ae_mansell"
+
+organizer_email: "organizers-toronto-2017@devopsdays.org" # Put your organizer email address here
+proposal_email: "proposals-toronto-2017@devopsdays.org" # Put your proposal email address here
+
+# List all of your sponsors here along with what level of sponsorship they have.
+# Check data/sponsors/ to use sponsors already added by others.
+sponsors:
+  - id: arresteddevops
```
Comment (phrawzty) [inline data/events/2017-toronto.yml]: For some reason this is in the default template. Is ADO a community sponsor for your event, or should this be removed? 😄

File: data/events/2017-toronto.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+startdate:  # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate:  # The end date of your event. Leave blank if you don't have a venue reserved yet.
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start:  # start accepting talk proposals.
+cfp_date_end:  # close your call for proposals.
+cfp_date_announce:  # inform proposers of status
+
+# Location
+#
+coordinates: "43.7000, -79.4000" # The corrodinates of your venue. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
```
Comment (phrawzty) [inline data/events/2017-toronto.yml]: This is somebody's house out near the Davisville station. 😆

File: data/events/2017-toronto.yml (hunk 1 above)
Comment (WhatsChrisDoing) [inline data/events/2017-toronto.yml]: I vaguely remember hearing/reading that they were happy to be a community sponsor for any event which is why I left it in. @bridgetkromhout @mattstratton am I just making things up?

File: data/events/2017-toronto.yml (hunk 2 above)
Comment (WhatsChrisDoing) [inline data/events/2017-toronto.yml]: Let me work on that. I thought I lifted it from last year's venue.

Comment (WhatsChrisDoing) [inline data/events/2017-toronto.yml]: Hmm. It is the coordinates from last year. Let me get the right ones.

Comment (WhatsChrisDoing) [inline data/events/2017-toronto.yml]: This is nominally the middle of Toronto. I'll adjust it to be closer to last year's event location.

File: data/events/2017-toronto.yml (hunk 1 above)
Comment (WhatsChrisDoing) [inline data/events/2017-toronto.yml]: I'll pull it for now.

**Comment by WhatsChrisDoing** [comment]
@phrawzty I think it should be good now. I appreciate you catching these mistakes.

**Pull request merged**